Updates/841_Saltgurka-Honor_Hold_Updates.sql

```diff
@@ -0,0 +1,1092 @@
+
+-- The infantry that's doing work around the town and the guard at the outhouse should not be carrying any weapons. Override with an empty equip_template
+DELETE FROM `creature_equip_template` WHERE `entry`=123;
+INSERT INTO `creature_equip_template` (`entry`,`equipentry1`,`equipentry2`,`equipentry3`) VALUES
+(123,0,0,0);
+UPDATE `creature` SET `equipment_id`=123 WHERE `guid` IN(57895,57889,58153,57942,58142,58149,58154,57904); -- Use an empty equip_template
+
+-- ----------------------------------------------------------
+-- Father Malgor Devidicus - Drink emote
+-- ----------------------------------------------------------
+UPDATE `creature_template` SET `AIName`='EventAI' WHERE `entry`=16825;
+
+-- ----------------------------------------------------------
+-- Honor Hold Defenders
+-- ----------------------------------------------------------
+
+-- Outhouse guard GUID: 57942
+-- Since this guard has a gossip menu it needs to have a separate template from the other guards (as far as I know). 
+-- So what I did was I gave this one the template of the two talking guards outside the destroyed keep, and I gave them the normal guard template.
+-- I see no reason why those two would need a separate template anyways, so I thought it was better to reuse this one than create a separate one for the outhouse guard.
+
+UPDATE `creature` SET `id`=16842 WHERE `guid` IN(72636,72637);
+UPDATE `creature` SET `id`=20513,`spawndist`=0,`MovementType`=2 WHERE `guid`=57942;
+UPDATE `creature_template` SET `GossipMenuId`=20026,`NpcFlags`=1 WHERE `entry`=20513;
+
+DELETE FROM `db_script_string` WHERE `entry` BETWEEN 2000000888 AND 2000000891;
+INSERT INTO `db_script_string` (`entry`,`content_default`,`emote`) VALUES 
+(2000000888,'Do ya want me to call a medic? Maybe a priest? It''s been over an hour, mate!',5),
+(2000000889,'So help me, I''m gonna count to ten and if yer not outta there by the time I reach 10, I''m comin'' in!',5),
+(2000000890,'What''s goin'' on in there?? For the love of the LIGHT, hurry it up!',5),
+(2000000891,'OY! You in there, this is official Honor Hold business yer holdin'' up!',5);
+
+DELETE FROM `dbscripts_on_creature_movement` WHERE `id` BETWEEN 5794201 AND 5794204;
+INSERT INTO `dbscripts_on_creature_movement` (`id`,`delay`,`command`,`datalong`,`dataint`,`comments`) VALUES
+(5794201,0,0,0,2000000888,'Outhouse Honor Hold Guard Text1'),
+(5794202,0,0,0,2000000889,'Outhouse Honor Hold Guard Text2'),
+(5794203,0,0,0,2000000890,'Outhouse Honor Hold Guard Text3'),
+(5794204,0,0,0,2000000891,'Outhouse Honor Hold Guard Text4');
+
+DELETE FROM `npc_text` WHERE `ID` = 15878;
+INSERT INTO `npc_text` (`ID`,`text0_0`,`prob0`,`em0_0`) VALUES
+(15878,'OY! Wait in line, would ya? Outland food doesn''t agree with me at all.',1,5); -- Exclamation emote
+
+DELETE FROM `gossip_menu` WHERE `entry`=20026;
+INSERT INTO `gossip_menu` (`entry`,`text_id`) VALUES
+(20026,15878);
+
+-- Pathing for  Entry: 16821 'TDB FORMAT' 
+SET @GUID := 57942;

```

Comment:
My current policy on adding new spawns, is relocate it to a safe range to avoid DB collisions. If there is a gap wide enough to insert all of them near each other go for it, but if there isnt, make sure all are near each other. Currently I am using 150000-150500 on voa, so anything around 151000 is safe and wont collide with CDB or WDB since they dont use that range.